CBOR/CBOR.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk='Microsoft.NET.Sdk'>
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0</TargetFrameworks>
+    <TargetFrameworks>netstandard1.0; net7.0</TargetFrameworks>
```

Comment:
Note that this adds a targetframework, as the recommended approach to get trimming annotations. I'm not sure if this ends up generating another dll in the nuget package but it shouldn't make a big difference whether it does or doesn't.

Reply:
This is acceptable for now.